Lib/http/server.py

```diff
@@ -1061,7 +1077,7 @@ class HTTPSDualStackServer(DualStackServerMixin, ThreadingHTTPSServer):
 
     ServerClass = HTTPSDualStackServer if args.tls_cert else HTTPDualStackServer
 
-    test(
+    return test(
```

Comment:
```suggestion
    test(
```

Reply:
Fixed in 5f1fb944d41ea6604b2bab7ce9afd91ac33edabf